browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2721,6 +2731,8 @@ L.CanvasTileLayer = L.Layer.extend({
 			y -= verticalOffset;
 		}
 
+		app.socket.inputEventType = "mouse";
```

Comment:
so we do not use value from core in the end?
I remember you wanted to add some callback

Reply:
good